lib/active_merchant/billing/gateways/mercado_pago.rb

```diff
@@ -114,6 +114,7 @@ def add_processing_mode(post, options)
         post[:processing_mode] = options[:processing_mode]
         post[:merchant_account_id] = options[:merchant_account_id] if options[:merchant_account_id]
         post[:payment_method_option_id] = options[:payment_method_option_id] if options[:payment_method_option_id]
+        post[:notification_url] = options[:notification_url] if options[:notification_url]
```

Comment:
Looking at [Mercado Pago's documentation for the notification URL](https://www.mercadopago.com.co/developers/es/guides/notifications/webhooks/), this field isn't at all related to the processing mode. Therefore, this field is not dependent on the `processing_mode` option being set, which is the requirement for adding the fields in this method.

This field can either be added directly to the `purchase_request`, etc. methods above or be added as its own new method and then that method can be added to the `purchase_request`, etc. methods above.